app/views/lessons/show.html.haml

```diff
@@ -34,7 +34,7 @@
     Sadly the whiteboards prior to 2012-11-07 are gone forever
 - elsif user_signed_in?
   -if @whiteboard == true
-    %iframe{src: "//pad.railsschool.org/p/rails-school-lesson-#{@lesson.id}", style: "width:100%;height:600px;border:1px solid #CCCCCC;"}
+    %iframe{src: "//pad.railsschool.org/p/rails-school-lesson-#{@lesson.id}?userName=#{current_user.formatted_user_name}", style: "width:100%;height:600px;border:1px solid #CCCCCC;"}
```

Comment:
Please use the designated view helper - https://github.com/rails-school/school/blob/3a55ebf11562c5494d73b2c1b3fb3baeb453f4f3/app/helpers/application_helper.rb#L2